apps/dashboard/src/components/runs-data-table.tsx

```
@@ -0,0 +1,341 @@
+
+type ExecutionStatus = "pending" | "processing" | "completed" | "failed";
+
+interface RunData {
+    id: string;
+    workflowName: string;
+    filename: string | null;
+    status: ExecutionStatus;
+    startedAt: string;
+    completedAt: string | null;
+    errorMessage?: string;
+    extractionResult?: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)